napalm/ios/utils/textfsm_templates/show_lldp_neighbors_detail.tpl

```diff
@@ -17,7 +17,8 @@ Start
   ^System Description: -> Description
   ^System Description\s*-\s*${REMOTE_SYSTEM_DESCRIPTION}
   ^System Capabilities\s*?[:-]\s+${REMOTE_SYSTEM_CAPAB}
-  ^Enabled Capabilities\s*?[:-]\s+${REMOTE_SYSTEM_ENABLE_CAPAB} -> Record
+  ^Enabled Capabilities\s*?[:-]\s+${REMOTE_SYSTEM_ENABLE_CAPAB}
+  ^\s+IP\:\s${REMPRT_SYSTEM_MGMT_IP} -> Record
```

Comment:
What is remprt and where is this variable used?

Reply:
Actually correct name is REMOTE_SYSTEM_MGMT_IP. I will correct.

This variable is being used to get remote system's management ip which is used to build a network topology. Once you have remote system's management ip, then you can log into it and find devices downstream.